ui/app/styles/core.scss

```diff
@@ -1,83 +1,82 @@
 // Start with Structure & Bulma variables as a foundation
-@import "./utils/colors";
-@import "./utils/spacing";
-@import "bulma/sass/utilities/initial-variables";
+@import './utils/colors';
+@import './utils/spacing';
+@import 'bulma/sass/utilities/initial-variables';
 
 // Override variables where appropriate
-@import "./utils/bulma_variables";
+@import './utils/bulma_variables';
 
 // Utils
-@import "./utils/mixins";
-@import "./utils/animations";
+@import './utils/mixins';
+@import './utils/animations';
 
 // Bring in the rest of Bulma
-@import "bulma/bulma";
-@import "bulma/switch";
-@import "bulma/bulma-radio-checkbox";
```

Comment:
This was the only change here - this was a vendor CSS file and importing it in sass caused a warning - I moved it to `ember-cli-build.js` and used `app.import` to add it to the vendor CSS file.